src/System.Security.Cryptography.Xml/tests/SignedXmlTest.cs

```diff
@@ -1552,11 +1573,25 @@ public void VerifyHMAC_HMACOutputLength_Invalid()
         }
 
         [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "https://github.com/dotnet/corefx/issues/18690")]
         public void SignedXmlUsesSha256ByDefault()
         {
             const string expectedSignatureMethod = "http://www.w3.org/2001/04/xmldsig-more#rsa-sha256";
             const string expectedDigestMethod = "http://www.w3.org/2001/04/xmlenc#sha256";
+            ValidateSignedXmlDefaultHashAlgorithms(expectedSignatureMethod, expectedDigestMethod);
+        }
 
+        [Fact]
+        [SkipOnTargetFramework(~TargetFrameworkMonikers.NetFramework, "https://github.com/dotnet/corefx/issues/18690")]
+        public void SignedXmlUsesSha1ByDefault()
```

Comment:
Maybe worth putting CoreFx in front of 256 and NetFx in front of this one, so the differential is more apparent in a log?